src/cfml/system/modules_app/system-commands/commands/printTable.cfc

```diff
@@ -73,6 +73,16 @@
  * ╚══════════════════════╧═══════════════════╝
  * {code}
  * .
+ *  The "filter" argument allows you to pass a SQL "where" statement to filter your data
```

Comment:
@scottsteibeck Doesn't this bit of help text need removed since we moved the "filter" functionality into the "sql" command?